datadogclient/datadog_client.go

```diff
@@ -129,32 +150,80 @@ func (c *Client) AddMetric(envelope *events.Envelope) {
 	c.metricPoints[key] = mVal
 }
 
-func (c *Client) PostMetrics() error {
+func (c *Client) addEvent(envelope *events.Envelope) {
+	title := fmt.Sprintf("%s: %d", envelope.GetError().GetSource(), envelope.GetError().GetCode())
+	event := Event{
+		Title:        title,
+		Text:         envelope.GetError().GetMessage(),
+		DateHappened: envelope.GetTimestamp() / int64(time.Second),
+		Tags:         parseTags(envelope),
```

Comment:
I do not know what the envelope looks like for these events. Will tags still be parsed in a sane way?